README.md

```diff
@@ -74,4 +74,4 @@ See the [LICENSE](https://github.com/monsieurbiz/Mbiz_InvisibleCaptcha/blob/mast
 
 ## Contributors
 
-- The people of Monsieur Biz <https://monsieurbiz.com>
\ No newline at end of file
+- The people of Monsieur Biz <https://monsieurbiz.com>
```

Comment:
Can you put back the empty last line to avoid this diff please?

Then I'll merge this PR 👍 

Thanks!

Reply:
ping @allanlaal